scripts/python/phoedf.py

```diff
@@ -92,15 +111,75 @@ def flatten_indices(mu, nu):
     self.Pr = None
     self.tau = None
     self.Tg = None
+    self.ucon = None
+    self.bcon = None
+    self.J = None
+    self.Hcov = None
+    self.E = None
+    self.F = None
+    self.P = None
```

Comment:
Along with fixing the indexing for the new output shapes here, I added some additional quantities that can be evaluated by `phoedf`

Reply:
@AstroBarker you may find some useful formulae in here for your Julia stuff